Help2guides (#2373)
openmicroscopy/ome-documentation: omero/sysadmins/restricted-admins.rst

Full administrators in OMERO can create new administrators with
restricted privileges using the OMERO.web interface, see the

> :help:`Facility managers section <example_facility_manager.html>` in our User guides.

Reviewer: ```suggestion
:help:`Web Interface: Managing Users <introduction/docs/group-user-management.html>` in our User guides.
```

This leads more directly to the target info.

  executed using OMERO.web or CLI.

- OMERO.web: allows viewing and downloading the data,

>   see :help:`Viewing Data <iviewer/docs/index.html>`.

Reviewer: ```suggestion
  see :help:`Viewing Data <iviewer/docs/index.html>` and :help:`Downloading Data <download/docs/index.html>`
```

existing containers owned by the new owner. Since OMERO 5.4.0,
OMERO.web enables Organizers with :term:`Write Data` privilege
to create new containers belonging to other users,

> see the :help:`OMERO.web in Data structure <introduction/docs/data-management.html>`

Reviewer: ```suggestion
see the :help:`Manage Images in Datasets/Projects <introduction/docs/data-management.html#manage-images-in-datasets-projects>`
```

> section of our Help documentation.

Reviewer: ```suggestion
section of our User guides.
```

  of ownership (possible only in CLI, will be addressed in the 5.4.x
  series). Creation of Projects, Datasets
  or Screens for other users in OMERO.web is possible since OMERO 5.4.0,

>   see :help:`Data structure (OMERO.web) <introduction/docs/data-management.html>`.

Reviewer: ```suggestion
  see the :help:`Manage Images in Datasets/Projects <introduction/docs/data-management.html#manage-images-in-datasets-projects>`
```
